### PR TITLE
fix: updated the logo options

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
continuation of #425

### CODE REVIEW

File: src/ansys_sphinx_theme/__init__.py [hunk 1]
```diff
@@ -442,26 +442,27 @@ def configure_theme_logo(app: Sphinx):
...
 
-    if not theme_options.get("logo"):
+    if not logo_option:
         theme_options["logo"] = pyansys_logo
 
-    if not isinstance(theme_options.get("logo"), str):
+    if isinstance(logo_option, str) and logo_option not in {"ansys", "pyansys", "no_logo"}:
         raise ValueError(
-            f"Invalid logo option: {theme_options.get('logo')}, The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
+            f"Invalid logo option: '{logo_option}'. Valid options are 'ansys', 'pyansys', or 'no_logo'."  # noqa: E501
```
Comment (Revathyvenugopal162) [inline src/ansys_sphinx_theme/__init__.py:452]:
```suggestion
            f"Invalid logo option: '{logo_option}'. The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
```

File: src/ansys_sphinx_theme/__init__.py [hunk 2]
```diff
@@ -442,26 +442,27 @@ def configure_theme_logo(app: Sphinx):
...
 
-    if not theme_options.get("logo"):
+    if not logo_option:
         theme_options["logo"] = pyansys_logo
 
-    if not isinstance(theme_options.get("logo"), str):
+    if isinstance(logo_option, str) and logo_option not in {"ansys", "pyansys", "no_logo"}:
         raise ValueError(
-            f"Invalid logo option: {theme_options.get('logo')}, The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
+            f"Invalid logo option: '{logo_option}'. The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
```
Comment (Revathyvenugopal162) [inline src/ansys_sphinx_theme/__init__.py:452]:
```suggestion
            f"Invalid logo option: '{logo_option}', The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
```

File: src/ansys_sphinx_theme/__init__.py [hunk 3]
```diff
@@ -442,26 +442,27 @@ def configure_theme_logo(app: Sphinx):
...
 
-    if not theme_options.get("logo"):
+    if not logo_option:
         theme_options["logo"] = pyansys_logo
 
-    if not isinstance(theme_options.get("logo"), str):
+    if isinstance(logo_option, str) and logo_option not in {"ansys", "pyansys", "no_logo"}:
         raise ValueError(
-            f"Invalid logo option: {theme_options.get('logo')}, The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
+            f"Invalid logo option: '{logo_option}', The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
```
Comment (SMoraisAnsys) [inline src/ansys_sphinx_theme/__init__.py:452]:
```suggestion
            f"Invalid logo option: '{logo_option}'. The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
```

File: src/ansys_sphinx_theme/__init__.py [hunk 4]
```diff
@@ -442,26 +442,27 @@ def configure_theme_logo(app: Sphinx):
...
 
-    logo = theme_options.get("logo")
+    logo = logo_option
 
     if logo == "ansys":
         theme_options["logo"] = ansys_logo
     elif logo == "pyansys":
         theme_options["logo"] = pyansys_logo
-    elif theme_options.get("logo") == "no_logo":
+    elif logo_option == "no_logo":
```
Comment (SMoraisAnsys) [inline src/ansys_sphinx_theme/__init__.py:461]:
```suggestion
    elif logo == "no_logo":
```

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, I left some minor comments

File: src/ansys_sphinx_theme/__init__.py (hunk 4 above)
Comment (Revathyvenugopal162) [reply src/ansys_sphinx_theme/__init__.py:459]: resolved in [af05675](https://github.com/ansys/ansys-sphinx-theme/pull/431/commits/af05675afa5d0468c15ea4c3346806ff03affe0c)

**Pull request merged**